source/_patterns/02-molecules/branding/demo/brandings.yaml

```diff
@@ -0,0 +1,3 @@
+url: 'http://0.0.0.0:8080/pl/'
+site_name: 'Site Name'
+site_slogan: 'A Clever Slogan'
```

Comment:
Install an editorconfig plugin to your IDE to automatically use our project config. Will auto-add line endings.